website/client/src/components/chat/autoComplete.vue

```diff
@@ -170,11 +163,26 @@ export default {
     this.grabUserNames();
   },
   methods: {
+    solveSearchResults (searchTextWip) {
```

Comment:
Is Wip for `Work in progress`?  Why not just `searchText`?

Reply:
cuz the text is being worked on at the moment (as opposed to the trailing end of the text that is dropped right b4 this method that isn't being worked on).

tbh, I would not name it search. probably I would name it `textWip`

Reply:
Yeah, might be a personal preference thing, I always try to put as little acronyms in my code as possible since they require more mental overhead to parse from the reader (and assume the reader knows about them in the first place).

Perhaps something along the lines of `textFilter` or `textQuery`?

Come to think of it `solve` is also a bit non-specific.  Perhaps `filterSearchResults` would be a better method name?

Reply:
Filter is weird. Query is more accurate, but still weird.

Search results is completely wrong.

I stuck with `textFocus` eventually.